### DISCUSSION TITLE
Users without certificates

### DISCUSSION THREAD

**Discussion by chrisburr** [DIRACGrid/diracx#17] [Extension requirements]
From the BiLDx meeting.

TODO: Add more description.

**Comment by marianne013** [comment]
We foresee all our users having certificates until at least 2025, but if the WLCG abandons user certificates (which I guess is likely)  and at the same time the WLCG decides to accept e.g. letsencrypt certificates for servers the UK CA might cease to exist (at the moment this is speculation). We already have a contingent of US users who are struggling to acquire grid compatible certificates, I assume this will only get worse.

**Comment by chrisburr** [comment]
In LHCb we can manage with requiring users to run `dirac-proxy-init` to upload a proxy at least once every 400 days. The only requirement is to make sure that `diracx login` fails in a clear way if the user doesn't currently have a proxy uploaded.

**Comment by iueda** [comment]
Belle II does not foresee any users without certificates for now. We are still using VOMS, not IAM yet.
Our transition from X.509 to tokens will happen only when all the Grid services in use would be accessible without X.509, that would take time.
